docs/user/index.rst

```diff
@@ -0,0 +1,45 @@
+User Guide
+==========
+
+This section of the documentation provides information about how to install
+uvloop as well as how to use uvloop.
+
+
+Installation
+------------
+
+`uvloop` is available from PyPI. It requires Python 3.5 and Cython.
+
+Use pip to install it.
+
+.. code-block:: console
+
+    $ pip install uvloop
+
+.. note::
+
+    Use the latest version of Cython to minimise the potential for build
+    issues.
```

Comment:
Ah, I see you've added the `requirements.txt` file. I think we only need `requirements.dev.txt`. Please use `pip freeze` to list concrete deps versions.